CONTRIBUTING.md

```diff
@@ -34,28 +34,26 @@ Tests are stored in the [tests](/tests) directory.
 
 Running tests locally:
 
-```
-make prepare-check && make check
-```
-
-The following packages are installed by 'make prepare-check'
-if they are not installed already:
+The following packages should be installed beforehand:
 
     python3-flexmock
     python3-requre
 
-Or you can run tests in a container:
-
-    make prepare-build
-    make build
-    make check-in-container
+```
+make check
+```
 
-The following packages are installed by 'make prepare-build'
-if they are not installed already:
+Or you can run tests in a container; the following packages should be
+installed beforehand:
 
     podman
     ansible-bender
 
+Then:
+
+    make build
+    make check-in-container
+
```

Comment:
Please adjust the wording of this part to resemble [the one here](https://github.com/packit/packit-service-fedmsg/pull/43/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R9-R30).